### PR TITLE
meta: the final migration

### PR DESCRIPTION
**Pull request by arthyn**
This concludes a 2 year stint of this desk being named incorrectly. Also renames agents to their more appropriate names and moves agents from %groups that should be in %landscape instead.

Depends on #163 to be released first so that data gets migrated before the other agents get shut down.

We'll need to yell from the rooftops @jamesacklin for third parties to know about all the moves which can be seen below: 

## desk
%garden -> %landscape

### renames
- %settings-store -> %settings

### from old groups
- %s3-store -> %storage

### from new groups
- %hark
- %contacts

### unchanged
- %docket
- %treaty
- %reel
- %bait

### final list
- %bait
- %contacts
- %docket
- %hark
- %reel
- %settings
- %storage
- %treaty

### CODE REVIEW

**Comment by ashelkovnykov** [comment]
See [these](https://github.com/tloncorp/landscape/issues/149#issuecomment-1533691452) [comments](https://github.com/tloncorp/landscape/issues/149#issuecomment-1533730354) on #149

File: desk/app/settings.hoon [hunk 1]
```diff
@@ -42,23 +52,36 @@
...
+          %put-entry   (put-entry:do [desk buc key val]:evt)
+          %del-entry   (del-entry:do [desk buc key]:evt)
+        ==
+      ::
+        ::  XX: deprecated; migration code
+          %noun
+        ?>  ?=(%migrate !<(%migrate vas))
+        =/  bas  /(scot %p our.bowl)/settings-store/(scot %da now.bowl)
+        :-  ~
+        ?.  .^(? %gu bas)
```
Comment (ashelkovnykov) [inline desk/app/settings.hoon:72]: Missing latest changes from #163

File: desk/app/storage.hoon [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    ?+  mark  (on-poke:def mark vase)
+        %storage-action
+      (poke-action !<(action vase))
+    ::
+      ::  XX: deprecated; migration code
+        %noun
+      ?>  ?=(%migrate !<(%migrate vase))
+      =/  bas  /(scot %p our.bowl)/s3-store/(scot %da now.bowl)
+      :-  ~
+      ?.  .^(? %gu bas)
```
Comment (ashelkovnykov) [inline desk/app/storage.hoon:82]: Same as above

File: desk/sur/groups.hoon [hunk 3]
```diff
@@ -0,0 +1,10 @@
+|%
+::  pulled from groups
+::
+::  $flag: ID for a group
++$  flag  (pair ship term)
+::
+::  $nest: ID for a channel, {app}/{ship}/{name}
++$  nest  (pair dude:gall flag)
+::
+--
```
Comment (ashelkovnykov) [inline desk/sur/groups.hoon:10]: nit: missing newline

**Review by ashelkovnykov — CHANGES_REQUESTED** [review]

**Review by ashelkovnykov — APPROVED** [review]
Seems fine to me. The ordering has been inverted and this branch is now attempting to merge into the one from #163, correct?

**Comment by arthyn** [comment]
right because the other is preceeding and I wanted to hide those changes here

**Comment by arthyn** [comment]
New fun and interesting changes 🤣

**Pull request merged**